roles/taginfo/files/10_download.sh

```diff
@@ -1,3 +1,20 @@
-mkdir -p /data/work/taginfo/data
-cd /data/work/taginfo/data
-wget -q -N http://download.openstreetmap.fr/extracts/europe/france.osm.pbf
+#! /bin/bash
+
+echo "$(date --rfc-3339=s) INFO: ${0} start ..."
+if [ -d /data/work/taginfo/data ]; then
+ echo "$(date --rfc-3339=s) DEBUG: dir ok"
+else
+ echo "$(date --rfc-3339=s) WARN: dir /data/work/taginfo/data inexistant. tentative mkdir"
+ mkdir -p /data/work/taginfo/data || { echo "$(date --rfc-3339=s) ERROR: mkdir" ; exit 1 ; }
+ echo "$(date --rfc-3339=s) INFO: mkdir ok"
+fi
+cd /data/work/taginfo/data || { echo "$(date --rfc-3339=s) ERROR: cd" ; exit 1 ; }
+echo "$(date --rfc-3339=s) INFO: ${0} pbf avant download"
+ls -l france.osm.pbf
+wget -N http://download.openstreetmap.fr/extracts/europe/france.osm.pbf || echo "$(date --rfc-3339=s) ERROR: wget"
+echo "$(date --rfc-3339=s) INFO: ${0} pbf apres download"
+ls -l france.osm.pbf
+echo "$(date --rfc-3339=s) INFO: ${0} start ..."
+wget -q -N http://download.openstreetmap.fr/extracts/europe/france.state.txt || echo "$(date --rfc-3339=s) ERROR: wget" 
```

Comment:
Il n'y a plus besoin du state.txt si tu utilises pyosmium-up-to-date pour la mise à jour.

Reply:
pyosmium-up-to-date n'est pas (encore) utilisé, c'est osmosis

Reply:
Ah oui, en effet.